frequency-chain/frequency-xyz
# Upgrade all packages

# Description

Simple package upgrade. Includes the upgrade to eslint v9

## Screenshot(s)

No changes

## Type of change

Please delete options that are not relevant.
- [x] CI/Dependencies/etc...

# How to Test?



1. Pull and run locally
2. See that nothing has changed.

# Checklist:
